Resource/PYTHON/multilateration/multilateration.py

```diff
@@ -0,0 +1,287 @@
+import math
+
+import numpy as np
+
+import multilateration.parameters as param
+
+# Initializing the variables maximum_time_diff and max_hydrophone_distance.
+# These values are updated in the function initialize_trilateration_globals
+
+# Initialized to -1 such that it's easy to check if the variables are incorrect
+
+max_hydrophone_distance = -1.0
+max_time_diff = -1.0
+
+# Functions for initializing
+
+
+def initialize_trilateration_globals():
+
+    # Calculating the distances between the hydrophones
+    dist_port_starboard = float(
+        math.sqrt(
+            math.pow(
+                param.HydrophoneDetails.STARBOARD_HYD_X
+                - param.HydrophoneDetails.PORT_HYD_X,
+                2,
+            )
+            + math.pow(
+                param.HydrophoneDetails.STARBOARD_HYD_Y
+                - param.HydrophoneDetails.PORT_HYD_Y,
+                2,
+            )
+            + math.pow(
+                param.HydrophoneDetails.STARBOARD_HYD_Z
+                - param.HydrophoneDetails.PORT_HYD_Z,
+                2,
+            )
+        )
+    )
```

Comment:
Extract this into a function, e.g.:
```python
def calculate_distance():
```
or use the position class with `abs()` that I have written in signal_generation.